discover/portal/articles/210-setting-up/01-system-settings/02-config-files/02-factory-configurations.markdown

```diff
@@ -0,0 +1,101 @@
+# Factory Configurations [](id=factory-configurations)
+
+Configurations supporting multiple independent occurrences of the same
```

Comment:
Maybe it's easier to understand if we say "Configuration supporting multiple entries are called *factory configurations*"? 
This is consistent with the UI which uses the term Entry already